src/shared.ts

```diff
@@ -46,7 +45,7 @@ namespace Logger {
   }
 }
 
-interface Logger extends Record<Logger.Type, Logger.Function> {}
+interface Logger extends Record<Logger.Type, Logger.Function> { }
```

Comment:
Why change irrelevant code?

Reply:
> Why change irrelevant code?

The formatting tool was changed by hand and has now been changed back.